cmp/grid/GridModel.js

```diff
@@ -145,6 +146,11 @@ export class GridModel extends HoistModel {
     /** @member {boolean} */
```

Comment:
for symmetry, let's just comment both of these properties, not define on in terms of the other.  Also, perhaps a changelog message